code/backend/batch/utilities/loggers/TokenLogger.py

```diff
@@ -1,24 +1,15 @@
 import logging
-from opencensus.ext.azure.log_exporter import AzureLogHandler
-from ..helpers.EnvHelper import EnvHelper
 
 
+# TODO: We probably don't need a separate class for this
```

Comment:
Can we raise another issue for this and I think this is the only code that needs APPINSIGHTS_CONNECTION_STRING - so can we get rid of that too? Happy for you to close this comment when we have a linked issue to clean these connection strings up (unless you have one already!)

Reply:
Raised #637 and #638